frontend/src/components/searchMetrics/GCSideBar.js

```diff
@@ -154,29 +154,31 @@ export default function SideBar(props) {
 	useEffect(() => {
 		try {
 			gameChangerAPI.gcOrgSealData().then(({data}) => {
-				let orgSources = data.filter((org) => org.image_link.startsWith('s3://'));
-				let folder = orgSources[0].image_link.split('/');
-				folder = folder[folder.length - 2];
-				const thumbnailList = orgSources.map(item => {
-					let filename = item.image_link.split('/').pop();
-					return {img_filename: filename}
-				});
-				gameChangerAPI.thumbnailStorageDownloadPOST(thumbnailList, folder, state.cloneData).then(({data}) => {
-					const buffers = data;
-					buffers.forEach((buf,idx) => {
-						if (buf.status === 'fulfilled') {
-							if (orgSources[idx].image_link.split('.').pop() === 'png'){
-								orgSources[idx].imgSrc = 'data:image/png;base64,'+ buf.value;
-							} else if (orgSources[idx].image_link.split('.').pop() === 'svg') {
-								orgSources[idx].imgSrc = 'data:image/svg+xml;base64,'+ buf.value;
+				let orgSources = []
+				if(data.length > 0){
+					orgSources = data.filter((org) => org.image_link.startsWith('s3://'));
+					let folder = orgSources[0].image_link.split('/');
```

Comment:
Is it possible for the previous s3 filter to result in no orgSources? If so then should just change the length check to `if (orgSources.length > 0).`

Reply:
you are absolutely correct, I'll update the logic